contents/blog/posthog-vs-logrocket.md

```diff
@@ -0,0 +1,775 @@
+            <td><strong>PostHog</strong></td>
+            <td></td>
+        </tr>
+    </thead>
+    <tbody>
+            <tr>
+            <td><strong>Event tracking</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Track manually instrumented events</td>
+        </tr>
+        <tr>
+            <td><strong>Autocapture</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Automatically track events without instrumentation</td>
+        </tr>
+        <tr>
+            <td><strong>Combined events</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Track related events as a single trackable action or behavior</td>
+        </tr>
+        <tr>
+            <td><strong>Reverse proxy available</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Send events from your own domain to capture more data</td>
+        </tr>
+        <tr>
+            <td><strong>Cross-domain tracking</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Track users across domains and sub-domains</td>
+        </tr>
+        <tr>
+            <td><strong>Server-side tracking</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Send events from your server</td>
+        </tr>
+        <tr>
+            <td><strong>Capture API</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Send events through an API</td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+> ### Should you autocapture events?
+>
+> LogRocket and PostHog both support manual instrumentation, as well as autocapture. Autocapture is much faster to setup, but some argue that it creates too much noise to be useful. We disagree, and it’s why PostHog gives you your first million events for free, every month — so you can capture events freely. [It’s something we feel quite strongly about](/blog/is-autocapture-still-bad). 
+
+### Library support
+
+<div className="overflow-x-auto -mx-5 px-5">
```

Comment:
This table is borked

Reply:
I tried fixing it, but couldn't get it to line up well. The best option (if you want to solve it) seems to be adding another column like the [posthog vs mixpanel post](https://posthog.com/blog/posthog-vs-mixpanel#client-libraries)